### PR TITLE
Update Rubocop and fix offenses

### PR DESCRIPTION
**Pull request by archanaserver**
Rubocop update to 1.28.0 and fix offenses.

### CODE REVIEW

File: .ruby-version [hunk 1]
```diff
@@ -0,0 +1 @@
+2.7.6
```
Comment (ehelms) [inline .ruby-version:1]: Where did this file come from? I think we should not include it. You may need to add it to the gitignore list.

Comment (archanaserver) [reply .ruby-version]: It gets committed by mistake, I've removed it!

File: Gemfile [hunk 2]
```diff
@@ -3,6 +3,12 @@ dev_gemfile = File.expand_path("Gemfile.dev.rb", __dir__)
 eval_gemfile(dev_gemfile) if File.exist?(dev_gemfile)
 gemspec
 
+group :rubocop do
+  gem 'rubocop', '~> 1.28.0'
+  gem 'rubocop-performance'
+  gem 'rubocop-rails'
```
Comment (ofedoren) [inline Gemfile:9]: Do we need rubocop-rails for hammer? :) Hammer and plugins are not dependent on Rails, so there is no need for that.

**Review by ofedoren — APPROVED** [review]
Since it's only rubocop related changes, LGTM

File: Gemfile (hunk 2 above)
Comment (archanaserver) [reply Gemfile]: @ofedoren it is updated now.

**Review by ofedoren — APPROVED** [review]
Thanks, @archanaserver, LGTM :)

**Pull request merged**